infra/cifuzz/cifuzz.py

```diff
@@ -71,12 +76,16 @@ def build_fuzzers(project_name, project_repo_name, commit_sha, git_workspace,
   build_repo_manager = repo_manager.RepoManager(inferred_url,
                                                 git_workspace,
                                                 repo_name=oss_fuzz_repo_name)
+  if not pr_ref and not commit_sha:
+    logging.error('A commit or pull request reference needs to be specified.')
```

Comment:
This is simply a programming error right?
Assert is probably appropriate in cases where you are just checking that you wrote something correctly.

Reply:
Fixed.